analyzers/its/expected/Ember-MM/Ember Media Manager-{9B57D3AB-AF12-4012-B945-284C2448DC81}-S2259.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I made a summary of the new found issues:
https://github.com/SonarSource/sonar-dotnet/pull/6054#issuecomment-1237186973